checker/manual/lock-checker.tex

```diff
@@ -290,19 +293,9 @@ \subsection{Polymorphic assignments\label{lock-type-checking-rules-polymorphic-a
     identical \<@GuardSatisfied(\emph{index})> type.
   \end{itemize}
 
-  Suppose that a method signature has multiple formal parameters with the
-  same \<@GuardSatisfied(index)> type.
```

Comment:
This paragraph was removed because this is expected behavior for polymorphic qualifiers.